### ISSUE TITLE
I6. “measurement”

### ISSUE THREAD

**Issue by congruili** [USGCRP/gcis-ontology#70]
Please create another term for “measurement” or find an existing term.  http://data.globalchange.gov/platform/topex-poseidon/instrument/topex-microwave-radiometer

**Comment by zednis** [comment]
There are many existing ontologies for describing measurements, but how we use the term 'measurement' is important - as it is a term with many different interpretations in usage.

Can you provide an example of how you would like to use 'measurement' in the instance data?

**Comment by justgo129** [comment]
Sure.  Here, seem solely to be using "measurement" to relate an "instrument_instance" (now just "instrument" in the turtle) to the datasets derived from it.  @rewolfe?

**Comment by rewolfe** [comment]
I'm concerned that if we focus on an instrument instance, we exclude the
more general case where a measurement is associated with a class of
instruments (e.g. AVHRR of which there are many, lots of in situ cases).
Measurement is also a general term that could be applied to a social or
economic survey (e.g. census).

On Fri, Jul 24, 2015 at 8:17 AM, justgo129 notifications@github.com wrote:

> Sure. Here, seem solely to be using "measurement" to relate an
> "instrument_instance" (now just "instrument" in the turtle) to the datasets
> derived from it. @rewolfe https://github.com/rewolfe?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/70#issuecomment-124501441
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
@rewolfe That makes a ton of sense.  Let's use a broader term then, relating an entity to the dataset produced from it.  Is there anything generic enough that we can use?

**Comment by justgo129** [comment]
How about gcis:producedDataset, or prov:generated?
gcis:producedDataset a owl:ObjectProperty ;
    rdfs:label "Produced Dataset" ;
    rdfs:comment "An activity generates datasets." ;
    rdfs:range gcis:Dataset ;
    rdfs:subPropertyOf prov:generated.

Contents copied from OWL file.
If we change "activity" to an entity or activity, we may be good to go.

**Comment by zednis** [comment]
It actually sounds like we talking about the relationship between agents and datasets, so I would use or extend `prov:wasAttributedTo` (entity to agent) or its inverse `prov:contributed` (agent to entity).

**Comment by justgo129** [comment]
If @bduggan , @aulenbac , or @rewolfe agree with this approach, I'll prepare the pull request using prov:contributed (agent to entity).

**Comment by zednis** [comment]
Oddly enough, we will want to add a definition of `prov:contributed` to the gcis ontology (in the prov namespace).  There was not agreement to formally define inverse properties for all RDF properties in PROV (because they we not formally defined in PROV-N) but the PROV-O team developed suggested property URIs for inverse properties.

`prov:wasAttributedTo` is formally defined.  `prov:contributed` is the PROV-O suggested URI for the inverse property, but this property is not formally defined in the PROV-O ontology.

...Yes, it is weird.

**Comment by justgo129** [comment]
@zednis please feel free to proceed with this one.  Although important, it's not a real high priority at this time.

**Comment by justgo129** [comment]
@zednis could you please provide a sample piece of turtle to reflect your suggestion that begins with "oddly enough?"  e.g. how would one or two of the appropriate lines for the turtle at:
http://data.globalchange.gov/platform/topex-poseidon/instrument/topex-microwave-radiometer
http://data.globalchange.gov/platform/topex-poseidon/instrument/topex-microwave-radiometer.thtml

look if implemented per your suggestion?

**Comment by zednis** [comment]
We would want to add a definition of `prov:contributed` to gcis.ttl:

```
prov:contributed
    a owl:ObjectProperty ;
    rdfs:subPropertyOf prov:influenced ;
    rdfs:label "contributed"@en ;
    rdfs:domain prov:Agent ;
    rdfs:range prov:Entity ;
    owl:inverseOf prov:wasAttributedTo .
```

We could then use `prov:contributed` in our instance data as if it had been defined in the PROV-O ontology.

**Comment by justgo129** [comment]
I think I understand.  So after this definition is added, a (poorly-written format-wise) sample turtle would read:
<http://data.globalchange.gov/instrument/topex-microwave-radiometer>
   a gcis:Instrument;
   prov:contributed <<http://data.globalchange.gov/dataset/nasa-podaac aviso_l4_dyn_topo_1deg_1mo>>;
prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-merged_tp_j1_ostm_ost_all_v2>;
....

?

**Comment by justgo129** [comment]
(with a URI in the first "prov:contributed")

**Comment by zednis** [comment]
Yes (except I am pretty sure the URI you are calling an instrument in your example would actually be have type `gcis:InstrumentType`).

**Comment by justgo129** [comment]
Yikes, I must have highlighted the wrong URL in the "I think I understand":

This was the one I meant.

I think I understand. So after this definition is added, a (poorly-written format-wise) sample turtle would read:
<http://data.globalchange.gov/platform/topex-poseidon/instrument/topex-microwave-radiometer.thtml>
a gcis:Instrument;
prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-aviso_l4_dyn_topo_1deg_1mo>;
prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-merged_tp_j1_ostm_ost_all_v2>;
....

?

**Comment by justgo129** [comment]
So yes, I think we're saying the same thing.  +1.

**Comment by zednis** [comment]
yep, this:

```
<http://data.globalchange.gov/platform/topex-poseidon/instrument/topex-microwave-radiometer>
   gcis:inPlatform <http://data.globalchange.gov/platform/topex-poseidon>;
   gcis:instrumentType <http://data.globalchange.gov/instrument/topex-microwave-radiometer>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-aviso_l4_dyn_topo_1deg_1mo>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-merged_tp_j1_ostm_ost_all_v2>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-merged_tp_j1_ostm_ost_cycles_v2>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-merged_tp_j1_ostm_ost_gmsl_ascii_v2>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac_mgdr>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-oscar_l4_oc_1deg>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-oscar_l4_oc_third-deg>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-recon_sea_level_ost_l4_v1>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex_altsdr>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex-altsdr>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex_gcp>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex-gdr>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex_l2_ost_gdr_vms>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex_l2_ost_mgdr_crossover>;
   prov:contributed <http://data.globalchange.gov/dataset/nasa-podaac-topex_l2_ost_tmr_replacement>;
   a gcis:Instrument .
```

**Comment by zednis** [comment]
@justgo129 I think our examples are in agreement.

**Comment by justgo129** [comment]
Works for me.  +1 for the addition of this predicate and subsequent change in the turtle template.  This also helps towards the resolution of #12.

**Comment by rewolfe** [comment]
Okay, this helps with my understanding of the other thread: #124.  +1

On Mon, Aug 10, 2015 at 2:57 PM, justgo129 notifications@github.com wrote:

> Works for me. +1 for the addition of this predicate and subsequent change
> in the turtle template. This also helps towards the resolution of #12
> https://github.com/USGCRP/gcis-ontology/issues/12.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/70#issuecomment-129564897
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
Excellent.  Closed #70 given the commit in the USGCRP branch, see:
https://github.com/USGCRP/gcis/blob/master/lib/Tuba/files/templates/instrument_instance/object.ttl.tut
